### DISCUSSION TITLE
File exclusions Scanner option

### DISCUSSION THREAD

**Discussion by reznoa** [gotson/komga#1811] [Ideas]
It would be nice to be able to specify files not to be scanned via file name.
For example, if it supports regular expressions, it would be nice to have an option to not scan files that match a pattern like "^.+_IsNotEpisode\.zip$".

**Comment by gotson** [comment]
why would you need that ?

**Reply by reznoa** [reply]
it's same as directory exclusions.
Directory exclusions option is needed to create directory for each series
but File exclusions option is not.

**Reply by gotson** [reply]
you only explain what it is, but not why you would need it. Can you provide a use case ?

**Reply by reznoa** [reply]
I have extra files for some series and I have them packed in a zip file.
Komga tries to parse those files as episode files and error report (Book does not contain any page).
I want to prevent this problem.

**Reply by gotson** [reply]
can you clarify what are those files maybe ?

**Reply by reznoa** [reply]
It is mostly a plain text file containing additional information about the series.

**Reply by gotson** [reply]
maybe don't put it in a zip, so Komga won't scan it

**Reply by reznoa** [reply]
that's bad solution
but i understand you

**Reply by gotson** [reply]
well i don't quite understand why you would put txt file in a zip, then it cannot be read directly without unzipping. Storing it in txt would make it available, and not generate errors in Komga. Seems like a good solution to me.
